### DISCUSSION TITLE
can we have additional community channel in irc and telegram?

### DISCUSSION THREAD

**Discussion by osevan** [speedb-io/speedb#318] [General]
For lightweight community setup,  we could communicate over irc and telegram too.

Discord is still for gamer and slack is to mainstream and not lightweight.

Thanks and

Best regards

**Comment by AmnonHanuhov** [comment]
@osevan Thanks for the advice, I think IRC could be a possibility in the near future. Being on more platforms is a good thing but that unfortunately also means that we (speedb members) need to keep track of more channels/platforms which is quite a bit of work, considering the fact that we want to supply the best support for the community. I get your point about why you (and others) would want an IRC channel, and we will definitely keep this option open and update you on our decision :)

**Comment by osevan** [comment]
https://github.com/RITlug/teleirc


Irc telegram bridge
